Add documentation for the RAD Manager

tobedeprez opened the pull request:
Updated RAD documentation and added documentation for the RAD Manager.

Still have to add documentation for the RAD API and add a link to it (cfr. TODO in docs). I didn't know how to do that. It concerns the messages under Skyline.DataMiner.Analytics.Rad in SLAnalyticsTypes (note that they inherit from deprecated messages with Mad in the name, but those can be ignored if possible).

MariekeGO commented:
Possibly the messages under Skyline.DataMiner.Analytics.Rad could be added somewhere in the [API documentation section](https://docs.dataminer.services/develop/api/index.html)? @PedroDebevere, what do you think?

SkylineDennisDr commented:
[someSmallComments.odt](https://github.com/user-attachments/files/19598948/someSmallComments.odt)

tobedeprez commented:
The following things have been changed

- Dennis' comments have been implemented
- Added RAD messages to the API documentation section (after discussion with @PedroDebevere) by adding the necessary types in the DataMiner.sln solution. 
- Added a general page with a description of the API (uid: RAD API). Not sure whether that page should be in the User Guide section (as it is currently), or it should be moved to the Developer Documentation section (I didn't immediately find a natural place there).

VeerleLedoux commented on user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Relational_Anomaly_Detection/RAD_API.md:
```diff
@@ -0,0 +1,105 @@
+
+# Working with the RAD API
+
+The *RAD API* is a set of messages that allows you to configure [Relational Anomaly Detection](xref:Relational_anomaly_detection) through an [automation script](xref:automation). Compared to using the [RAD Manager](xref:RAD_manager), the *RAD API* offers more flexibility, but comes with a steeper learning curve. It is particularly useful when your use case deviates from the default options provid ...
```
I would drop the deep  learning curve thing and write something like : Compared to using the RAD Manager, the RAD API offers more flexibility making it particularly useful for use case that  deviate from the default options provided by the RAD Manager.

VeerleLedoux commented on user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Relational_Anomaly_Detection/RAD_API.md:
```diff
@@ -0,0 +1,105 @@
+| Message | Response message | Description |
+|------|-----|-------------|
+| [AddRADParameterGroupMessage](xref:Skyline.DataMiner.Analytics.Rad.AddRADParameterGroupMessage) | [AddRADParameterGroupResponseMessage](xref:Skyline.DataMiner.Analytics.Rad.AddRADParameterGroupResponseMessage) | Adds a new parameter group to the RAD configuration or overwrites an existing one. See [Options for parameter groups](xref:Relational_anomaly_detection#options-for-parameter-groups) for ...
+| [GetRADDataMessage](xref:Skyline.DataMiner.Analytics.Rad.GetRADDataMessage) |[GetRADDataResponseMessage](xref:Skyline.DataMiner.Analytics.Rad.GetRADDataResponseMessage) | Retrieves historical anomaly scores for a specified group within a given time range. |
```
maybe write parameter group instead of just group, as you seem to do it for all other messages
